cypress/integration/tests_e2e/testWithPageObject.sepc.js

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
I think there's a typo in file name, it should be "spec.js" probably.

Reply:
My mistake ;-)

Reply:
Let's try to not put `Page Object Pattern/Page Object` into a filename or content. It's just a pattern.